src/plugins/intel_gpu/include/intel_gpu/graph/fused_primitive_desc.hpp

```diff
@@ -21,7 +21,7 @@ class NodeFuseParams {
 };
 
 struct fused_primitive_desc {
-    explicit fused_primitive_desc(std::shared_ptr<const primitive> prim) : desc(prim) {}
+    explicit fused_primitive_desc(std::shared_ptr<const primitive> prim) : desc(std::move(prim)) {}
```

Comment:
To reduce the number of `std::move`s and improve code readability I'd suggest to use passing by const reference instead of passing by value in all cases where it is possible. And then we will have only single copy c-tr call (as when `std::move` using) and better code design to my mind
```suggestion
    explicit fused_primitive_desc(const std::shared_ptr<const primitive>& prim) : desc(prim) {}
```